src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "1.0.0-rc.2"
+__version__ = "0.1.0"
```

Comment:
Is this version regression intended?

Reply:
This change was unintentional. Addressed here https://github.com/cisagov/windows-commando-vm-packer/pull/20/commits/fca2728ac8fdb8298991237ff2c1d66dc921be17#diff-ac165679b905b686c09d6d19d76ebd7f43ca3b811aeee571b378b68afc6fd696R1